src/Appium.Net/Appium/ScreenRecording/BaseStartScreenRecordingOptions.cs

```diff
@@ -11,7 +13,8 @@ public abstract class BaseStartScreenRecordingOptions<T> : BaseScreenRecordingOp
         /// <returns>self instance for chaining.</returns>
         public T WithTimeLimit(TimeSpan timeLimit)
         {
-            Parameters["timeLimit"] = (int) timeLimit.TotalSeconds;
+            Parameters["timeLimit"] = timeLimit.TotalSeconds
+                .ToString(CultureInfo.InvariantCulture);
```

Comment:
is it necessary to convert it to string?

Reply:
Not sure if it's necessary but all the documentation I saw stated string.

Reply:
I assume the server supports both string and numbers, but it makes more sense to keep it as int for me

Reply:
Reverted